src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -39,76 +39,87 @@
 					@scroll="debounceHandleScroll">
 					<NcAppNavigationCaption :class="{'hidden-visually': !isSearching}"
 						:title="t('spreed', 'Conversations')" />
-					<Conversation v-for="item of conversationsList"
-						:key="item.id"
-						:ref="`conversation-${item.token}`"
-						:item="item" />
-					<template v-if="!initialisedConversations">
-						<LoadingPlaceholder type="conversations" />
+
+					<!-- Initial Loading -->
+					<LoadingPlaceholder v-if="!initialisedConversations" type="conversations" />
+
+					<!-- Conversations List -->
+					<template v-if="!isSearching">
+						<Conversation v-for="item of conversationsList"
+							:key="item.id"
+							:ref="`conversation-${item.token}`"
+							:item="item" />
 					</template>
-					<Hint v-else-if="searchText && !conversationsList.length"
-						:hint="t('spreed', 'No matches')" />
-					<template v-if="isSearching">
-						<template v-if="!listedConversationsLoading && searchResultsListedConversations.length > 0">
+
+					<!-- Search results -->
+					<template v-else-if="isSearching">
```

Comment:
```suggestion
					<template v-else>
```
Will it make sense here?

Reply:
With large `<template>` blocks I wanted it to be more explicitly visible that this block is searching (not only by manual comments but also by the code).

But yep, it can be safely replaced with `v-else`